### PR TITLE
A bit ot refactoing

### PR DESCRIPTION
**Pull request by vugman**

### CODE REVIEW

File: src/components/Method/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Index = ({ method: { title, summary } }) => (
```
Comment (itamark) [inline src/components/Method/index.js]: Can we change `Index` const to `Method` here so it's clearer?

File: src/components/Calendar/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import DateDisplay from '../DateDisplay';
+import Method from '../Method';
+import methods from '../../data/methods';
+import { getDay, incrementDate, decrementDate } from '../../helpers/dateHelper';
+import './style.css';
+
+export default class Index extends Component {
```
Comment (itamark) [inline src/components/Calendar/index.js]: Maybe we should change the class name to `Calendar`?

**Review by itamark — CHANGES_REQUESTED** [review]
Just a few renamings 😃

File: src/components/Method/index.js (hunk 1 above)
Comment (vugman) [reply src/components/Method/index.js]: Oh damn my bad, of course!

File: src/components/Calendar/index.js (hunk 2 above)
Comment (vugman) [reply src/components/Calendar/index.js]: Same, didn't notice :-) good catch!

**Review by itamark — APPROVED** [review]

**Pull request merged**